Resources/views/macros/data-field-type.html.twig

```diff
@@ -637,6 +637,35 @@
 	{{ _self.messages(dataField) }}
 {% endblock %}
 
+
+
+{% block EMS_CoreBundle_Form_DataField_JsonMenuEditorFieldType %}
+	{% set doCompare = compare and dataField.rawData is not same as( compareRawData|get_string(dataField.fieldType.name) )  %}
+	{% set color = not doCompare ? '' : not dataField.rawData ? 'bg-red' : not compareRawData|get_string(dataField.fieldType.name) ? 'bg-green' : 'bg-orange' %}
```

Comment:
Color seems business logic for php side. Okay color is frontend. But that is css issue. Determining the class that has to be used is a business case, for PHP

Reply:
I' dont see how we can handle this.

Reply:
Maybe extend the FieldType interface (renderCompare ?) acting like a mini controller? (another pull request)